parsing/location.ml

```diff
@@ -242,7 +350,8 @@ let highlight_dumb ~print_chars ppf lb locs =
   end
 
 let show_code_at_location ppf lb locs =
-  highlight_dumb ~print_chars:false ppf lb locs
+  try highlight_dumb ~print_chars:false ppf lb locs
+  with Exit -> ()
```

Comment:
I think I overlooked something: who raises `Exit`?

Reply:
`highlight_dumb` might raise `Exit` (this is not changed by this patch, it was already possible). Apparently this does not happen currently, but I added this safeguard just in case.

Reply:
I had indeed missed it. :)
The change seems good then.